### PR TITLE
removed unused code

### PR DESCRIPTION
**Pull request by cryptoAtwill**
## Changes

- Removed unused code such as `randomness`, `runtime policy`.
- Migrated `tcid` to `primitives` and added `TAddress` crate
- Added default exports in `src/lib.rs`.
- Add readme.md to all crates.

## Tests

```
cargo build && cargo test
```

### CODE REVIEW

File: runtime/src/runtime/fvm.rs [hunk 1]
```diff
@@ -12,7 +12,6 @@ use fvm_shared::crypto::signature::Signature;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::error::{ErrorNumber, ExitCode};
 use fvm_shared::piece::PieceInfo;
-use fvm_shared::randomness::Randomness;
 use fvm_shared::sector::{
```
Comment (adlrocha) [inline runtime/src/runtime/fvm.rs:15]: All the types that implement sector or storage-specific functions can also be removed.

File: runtime/src/runtime/mod.rs [hunk 2]
```diff
@@ -32,12 +29,9 @@ pub mod fvm;
 #[cfg(feature = "fil-actor")]
 mod actor_blockstore;
 
-pub mod policy;
-mod randomness;
-
 /// Runtime is the VM's internal runtime object.
 /// this is everything that is accessible to actors, beyond parameters.
-pub trait Runtime<BS: Blockstore>: Primitives + Verifier + RuntimePolicy {
+pub trait Runtime<BS: Blockstore>: Primitives + Verifier {
```
Comment (adlrocha) [inline runtime/src/runtime/mod.rs:34]: `Verifier` can also be removed.

File: runtime/src/test_utils.rs [hunk 3]
```diff
@@ -21,7 +21,6 @@ use fvm_shared::crypto::signature::Signature;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::error::ExitCode;
 use fvm_shared::piece::PieceInfo;
-use fvm_shared::randomness::Randomness;
 use fvm_shared::sector::{
```
Comment (adlrocha) [inline runtime/src/test_utils.rs:24]: Same as above, these can probably be removed.

File: runtime/src/test_utils.rs [hunk 4]
```diff
@@ -632,39 +604,17 @@ impl MockRuntime {
...
-        entropy: Vec<u8>,
-        out: Randomness,
-    ) {
-        let a = ExpectRandomness {
-            tag,
-            epoch,
-            entropy,
-            out,
-        };
+    pub fn expect_get_randomness_from_beacon(&mut self) {
```
Comment (adlrocha) [inline runtime/src/test_utils.rs:616]: Why do we need to keep these? It is imposed by the trait definition? If this is the case, which one? I feel we can remove this from the runtime.

**Review by adlrocha — APPROVED** [review]
LGTM.

A few things before you merge:
- Consider adding also a brief description of what is included as part of the `fvm_utils` in the main `README` of the repo.
- All storage-specific code (like `Verifier` and `sector`-related features) can also be removed.
- Consider removing functions like `expect_randomness` from the runtime trait definition if needed.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,10 @@
 # fvm-utils
-This repo contains a series of crates that serve as utils for fvm develop and testing.
\ No newline at end of file
+This repo contains a series of crates that serve as utils for fvm develop and testing.
```
Comment (adlrocha) [inline README.md:2]:
```suggestion
This repo contains a series of crates that serve as utils for fvm development and testing.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,10 @@
...
+Here are some simple breakdowns:
+- runtime: Contains the runtime wrapper for communicating with `fvm`. It provides some
```
Comment (adlrocha) [inline README.md:4]: TODO: Add an extended description of the runtime. Do you mind adding an issue to remind ourselves to do it in the future (I can probably take this one)

Comment (cryptoAtwill) [reply README.md:4]: created, https://github.com/consensus-shipyard/fvm-utils/issues/4

**Pull request merged**